package-lock.json

```diff
@@ -0,0 +1,14020 @@
+{
```

Comment:
Should this file be included? We haven't been able to figure out a way to get npm5/node 8 deployed yet, but not sure if this will cause issues or not 🤔